make-rules/shared-macros.mk

```diff
@@ -578,7 +578,7 @@ export CCACHE := $(shell \
         fi; \
     fi)
 
-GCC_VERSION =	6
+GCC_VERSION =	7
```

Comment:
So, it's not only about dropping 32 Bit but also change to gcc 7?

Reply:
It was a convenience for rebuilding here, I remove it before merge.